Dockerfile

```diff
@@ -74,6 +74,7 @@ ARG RUBY_INSTALL_VERSION=0.8.3
 ARG RUBYGEMS_SYSTEM_VERSION=3.3.22
 
 ARG BUNDLER_V1_VERSION=1.17.3
+# When bumping Bundler, need to also regenerate `updater/Gemfile.lock` via `bundle update --bundler`
```

Comment:
This command resulted in bumping a bunch of lines beyond the Bundler version... I wasn't sure if that's because those are what our current Gemfile's resolve to, or because those happen to be the newest versions on RubyGems and regardless of whether we've upgraded Bundler still tries to pull those in? IE, is there a command that _only_ regenerates w/o pulling in new versions of other stuff?

Also, I added a code comment because I couldn't come up with a simple programmatic way to trigger regenerating the lockfile whenever we bump Bundler... since we can't do it in the `Dockerfiler.updater` build step since then it never makes its way back to the git repo. And we don't want to do it in the `Dockerfile` because that (purposefully) doesn't mount `/updater` code. 

We also (AFAIK) don't have a check that fails the build if any of the Dockerfile builds result in an unexpected `git diff`... that'd probably be useful to add so that we aren't silently having code mutate under our feet before deploy w/o us realizing it. That would have caught this problem actually.

Reply:
I think the reason is as @jurre pointed out. These dependencies are pulled from the sibling `dependabot-*` folders, because they are path dependencies. This kind of dependencies are never locked and always use what's there in the filesystem.